starter-code/javascript/main.js

```diff
@@ -52,22 +54,36 @@ function setResetBtn() {
   btnRight.innerHTML = "RESET";
   btnRight.className = "btn reset";
 }
-
+function setMinSec() {
+  minSec.innerHTML = chronometer.setTime()
+}
 // Start/Stop Button
 btnLeft.addEventListener('click', function () {
-    if (chronometer.currentTime === 0){
-      setStopBtn();
-      setSplitBtn();
-      chronometer.startClick();
-    }
-    else{
-      chronometer.stopClick();
-      setStartBtn();
-      setResetBtn();
-    }
+  
+  if (chronometer.currentTime === 0){
```

Comment:
Lo que hacéis es que solo se ejecute cuando el tiempo es 0, por tanto no podéis volver a ejecutar si lo paráis. Una forma para solucionar este problema es: 
`if(btnLeft.innerHTML === "START")` Miráis que el innerHTML tenga el valor de **START** y así podéis volver a ejecutarlo.